torchinductor/decomposition.py

```diff
@@ -278,11 +280,6 @@ def scatter_reduce(self, dim: int, index, src, reduction_type, **kwargs):
     return self.clone().scatter_reduce_(dim, index, src, reduction_type, **kwargs)
 
 
-@register_decomposition([aten.narrow])
```

Comment:
Did you mean to remove this?  I don't see a replacement decomp.

Reply:
Yeah these are both decompositions for "CompositeImplicitAutograd" ops that are no longer needed now that we always decompose CompositeImplicitAutograd ops (i.e. Ed's change).

https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/native_functions.yaml#L3749

https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/native_functions.yaml#L5411